text/0000-artifact-hub-integration.md

```diff
@@ -0,0 +1,339 @@
+│   │   └── ...
+│   ├── ruby
+│   │   ├── 0.4.0
+│   │   │   └── artifacthub-pkg.yml
+│   │   │   └── readme.md
+│   │   └── ...
+│   └── ...
+└── ...
+```
+
+The CNB Artifact Hub organization will register the existing buildpack packages in Artifact Hub by defining Artifact Hub
+registries with URLs that point to packages in the following pattern:
+
+- https://github.com/buildpacks/registry-index/packages/heroku/dotnet
+- https://github.com/buildpacks/registry-index/packages/heroku/java
+- https://github.com/buildpacks/registry-index/packages/initializ-buildpacks/go
+- https://github.com/buildpacks/registry-index/packages/paketo/python
+- https://github.com/buildpacks/registry-index/packages/paketo/ruby
+- ...
+
+> [!WARNING]
+> At this stage, the buildpack registry API still uses the CNB registry index as its source. This means that `pack` is
+> not pulling from ArtifactHub.io. Publishers should not change their publishing method at this time. There could be
+> confusion if buildpack artifacts exist in ArtifactHub.io but not the registry index, because `pack` only knows about
+> the packages defined in the index.
+
+### Requirement 5 - Update the buildpacks [registry indexer](https://github.com/buildpacks/registry-api/blob/main/cmd/index-buildpacks/main.go) to use to ArtifactHub.io instead of directly from the index file structure.
+
+_The goal here is that the registry service no longer crawls the packages inside the buildpack index project. It will
+use
+ArtifactHub.io as the source of truth (ArtifactHub.io will use the registry index as its source of truth)._
+
+At this point, ArtifactHub.io will be scanning the registry index using the new file structure outlined in the previous
+step. Instead of the CNB registry service scanning this as well, the registry service will populate its database based
+on calls to the ArtifactHub.io API.
+
+When this step is complete, publishers will be able to take full owership of their publishing process. The buildpacks
+registry index will continue to be read/write for 12 months, so publishers can continue to publish to the buildpacks
+registry.
+
+At this point, publishers have the option of claiming ownership of their packages and publishing to ArtifactHub.io only.
+The CNB registry will read any new buildpacks that are published to ArtifactHub.io outside the CNB registry index
+publishing process because it is calling the ArtifactHub.io API.
+
+`pack` will continue reading from the CNB registry API.
+
+At this point, publishers can publish to either the buildpack registry index or ArtifactHub.io. By the end of 12 months,
+publishers should only use the ArtifactHub.io method for publishing, because the buildpack registry index will become
+readonly at this point and will not accept new publishes.
+
+### Requirement 6 - Update `pack` to pull from ArtifactHub.io
+
+We will introduce a new flag to `pack` called `registry-schema`. The options for this flag are `cnb` and `artifacthub`.
```

Comment:
instead of `cnb`, i think we should use `legacy` or something that strongly indicates you don't want to favor it. The `cnb` value kinda implies it's the preferred one